source/acquaman/SGM/SGMXASScanController.cpp

```diff
@@ -24,10 +27,15 @@ SGMXASScanController::~SGMXASScanController()
 
 void SGMXASScanController::onAxisFinished()
 {
+	QTime startTime = QTime::currentTime();
+	QTime stepTime  = QTime::currentTime();
+	int currentStep = 1;
 	// STEP 1: Data Checks & Meta Info Collection
 	if(!generateAnalysisMetaInfo())
 		return;
 	// END OF STEP 1
+	qDebug() << QString("[Step %1] Total Time: %2 Delta Time %3").arg(currentStep++).arg(startTime.msecsTo(QTime::currentTime())).arg(stepTime.msecsTo(QTime::currentTime()));
```

Comment:
Ha! Laughably yes. Apparently I couldn't take the 15 seconds to remove those ... embarrassing.